Makefile

```diff
@@ -47,11 +47,11 @@ spellcheck:  ## Runs a spellchecker over all code and documentation
 
 .PHONY: test
 test: lint ## Run tests
-	pytest tests/flytekit/unit
+	pytest tests/flytekit/unit tests/flytekit_compatibility
 
 .PHONY: unit_test
 unit_test:
-	pytest tests/flytekit/unit
+	pytest tests/flytekit/unit tests/flytekit_compatibility
```

Comment:
just to be safe, should we run the tests twice (with the flag set to false and true)?

Reply:
not sure i understand... what do you mean?

Reply:
sorry, I mean run the tests once where the env var `FLYTE_SDK_USE_STRUCTURED_DATASET` is set to `TRUE` and again with it set to false (or possibly unset?)

Reply:
this is pretty hard to do...  i can copy/paste some tests into the compatibility folder.